engine/src/main/java/com/agiletec/aps/system/RequestContext.java

```diff
@@ -53,14 +54,22 @@ public Object getExtraParam(String name) {
 	 * @param name Il nome del parametro
 	 * @param param Il parametro da aggiungere
 	 */
-	public void addExtraParam(String name, Object param) {
+	public void addExtraParam(String name, Serializable param) {
 		if (isCurrentFrameOrCurrentWidget(name)) {
 			EntThreadLocal.set(name, param);
 		} else {
 			this._extraParams.put(name, param);
 		}
 	}
 
+	public void setExecutorBeanContainer(ExecutorBeanContainer param) {
```

Comment:
we should add an annotation @Deprecated ad a link to the problem / ticket to explain the problem / solution

Reply:
I added a warning here and in the ExecutorBeanContainer class. Please have a look, thanks.